modules/ROOT/pages/configure-ingress-http-resource.adoc

```diff
@@ -0,0 +1,252 @@
+
+----
+apiVersion: rtf.mulesoft.com/v1
+kind: HttpRouteTemplate
+metadata:
+  name: k8s-gateway-api-cr
+  namespace: rtf
+spec:
+  baseEndpoints:
+    - https://demo.example.com
+  resources: |-
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: {{ .ResourceName }}
+  namespace: {{ .Namespace }}
+spec:
+  parentRefs:
+  - kind: Gateway
+    name: cluster-gateway
+  hostnames:
+  - {{ .Host }}
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: {{ .Path }}
+    backendRefs:
+    - name: {{ .Service.Name }}
+      port: {{ .Service.Port }}
+
+----
+
+[NOTE]
+You are responsible for the configuration of the Kuberneates Gateway resources and allowing HTTP Routes from the route's namespace to be attached.
+
+
+== Using TLS in the HTTP Route-Template Custom Resource
+
+To use TLS in the routing resources in an HTTP route template, create a TLS secret in the Runtime Fabric namespace. For details, refer to the https://kubernetes.io/docs/concepts/services-networking/ingress/#tls[Kubernetes TLS] documentation.
+
+To configure the Runtime Fabric to manage the secrets lifecycle (Create, Update, and Delete) across all application namespace, use the `rtf.mulesoft.com/synchronized: "true"` label on the TLS secret. For example:
+
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: testsecret-tls
+  namespace: <agent-namespace>
+  labels:
+    rtf.mulesoft.com/synchronized: "true"
+data:
+  tls.crt: base64 encoded cert
+  tls.key: base64 encoded key
+type: kubernetes.io/tls
+
+----
+
+Using this label is highly recommended for secrets management. If you don’t add this label, you must manually update the secret in each namespace.
+
+== Modify an Existing HTTP Route-Template CRD
+
+You can modify existing `httproutetemplates` templates even if they are used for application deployments. Note that previously created HTTP routing resources based on this template aren’t updated if you modify the template.
+
+To update existing HTTP routing resources with the template changes, perform a redeployment of the Mule app from the Anypoint control plane. For example, if you update the base domain, which requires changing the endpoint for the ingress template, remove the old endpoint and add the new one as part of the application redeployment.
+
+== Frequently Asked Questions
+
+*Does Runtime Fabric support the existing ingress templates?*
+
+Yes, Runtime Fabric supports both old and new templates. However, use the new Custom Route templates to configure the ingress for your Mule application deployments.
+
+*What happens if there is a collision between an ingress template and an HTTP Route CR?*
+
+The new HTTP Route CR takes precedence, and the old ingress is used as a fallback.
+
+*What happens if there is a BaseEndpoint or URL collision between two HTTP Route CRs?*
+
+The HTTP Route CR with the name that comes first alphabetically is used for rendering.
```

Comment:
"for rendering". This is a new term not used on this page. We've previously said "applied". 

"The HTTP Route CR with the name that comes first alphabetically is applied"